_posts/2020-07-19-stage-set-for-knockouts.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: The stage is set for the SR2020 Knockouts
+---
+
+After two weekends and 48 matches, the league stage of the SR2020 Virtual
+Competition is complete, setting the stage for the knockouts and the grand final
+next weekend.
+
+This weekend saw strong performances from a number of teams, including a number
+of firsts: Kenilworth Sixth Form collected 5 tokens into their scoring zone in
+one match and both Hills Road Sixth Form College's first team and JAMDynamics
+stealing tokens from each others scoring zones.
+
+Next weekend the teams will compete in the knockouts, which is the final test
+for the teams' months of coding preparation. Once again the teams have a week to
+work on and improve their robots before the 10am BST deadline to submit their
+code ahead of the next set of matches.
+
+The robots which perform best will make it through the Quarter-Finals to the
+Semi-Finals and then to the grand Final of Student Robotics 2020. It remains
+possible for any team to become the overall winners -- while the league scores
+seed the knockouts, making progression easier for robots who scored well so far,
+all teams will compete in the first round of the knockouts.
+
+As well as prizes for the 1st, 2nd and 3rd places, teams are also competing
+for a number of other awards, including the Committee Award (given to the team
+that displays the most extraordinary ingenuity in their robot), and the Online
+Presence Award (for the team that is judged to have the best online presence).
+Full details of these awards and those for earlier stages of the competition are
+available in our [rulebook](/docs/resources/2020/rulebook.pdf)
+
+If you would like to catch up on this weekend's livestreamed matches, they're
+available on YouTube:
+
+* [Saturday: League Session 1](https://youtu.be/symUE1E4niI)
```

Comment:
Looks like the link text needs updating here.

Reply:
625e15d reworks this somewhat.

Reply:
625e15d reworks this somewhat.